content/docs/ops/setup/injection/index.md

```diff
@@ -11,14 +11,19 @@ time. Injection can be scoped to particular sets of namespaces using
 the webhooks `namespaceSelector` mechanism. Injection can also be
 enabled and disabled per-pod with an annotation.
 
-Whether or not a sidecar is injected is depends on three things:
+Whether or not a sidecar is injected depends on three pieces of configuration and two limitations:
```

Comment:
I would not position is as 2 limitations... it is something by design and don't want to confuse users.

Reply:
instead of two limitations, you might try `two security rules`.